bubble_sort/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+)
+
+func sweep(numbers []int) {
+	numbersLength := len(numbers)
+	firstIndex := 0
+	secondIndex := 1
+
+	for secondIndex < numbersLength {
+		firstNumber := numbers[firstIndex]
+		secondNumber := numbers[secondIndex]
+		if firstNumber > secondNumber {
+			numbers[firstIndex] = secondNumber
+			numbers[secondIndex] = firstNumber
+		}
+
+		firstIndex++
+		secondIndex++
+	}
+}
```

Comment:
1. I don't think we need the prefix/suffix 'number' in all those places, `length`, `first`, `second` would be find. 
2. You don't need 2 indexes, you could just go with `i`, `i+1`
3. How about iterating with `for i := 0; i < len(numbers) - 1; i++`? Or even better, with `range`?